src/configuration.ts

```diff
@@ -19,6 +19,13 @@ import * as fs from "fs";
 import * as path from "path";
 import * as findup from "findup-sync";
 
+import {arrayify, objectify} from "./utils";
+
+export interface IConfigFile {
```

Comment:
Sorry to chime in, but I think it should be renamed is `IConfigurationFile` as this interface is useful to extensions.